psiturk/psiturk_shell.py

```diff
@@ -94,7 +94,7 @@ def amt_services_wrapper(self):
                     '- `psiturk server off`',
                     '- `psiturk debug -p`'])
                 message = '{}{}'.format(e.message, still_can_do)
-                self.pfeedback(message)
+                self.perror(message)
```

Comment:
Setting feedback_to_output = True caused another test to fail. Changing this to `perror` fixed. `pwarning` is also probably an option.

Reply:
I'm okay with this being perror, conceptually

Reply:
You never got `pytest` or `tox` working locally? `tox` is a good bet when pytest is misbehaving, because tox creates an isolated testing environment. I don't mind you pushing code to get tests to run, but local testing would be much faster development cycle for you.

Reply:
Pytest is working for me. 

It wasn't before. I eventually worked out that the development environment I was using had a non-canonical set of dependencies installed via pip; I forget exactly why. Creating new environments from scratch got it working fine. So I created one for py36, py37 and py38. 

Yeah, testing locally is faster for sure.

I'll try tox.